packages/node_modules/nav-frontend-inngang/src/appinngang.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import PT from 'prop-types';
+import { Innholdstittel } from 'nav-frontend-typografi';
+
+export default class AppInngang extends Component { // eslint-disable-line react/prefer-stateless-function
+    render() {
+        return (
+            <div {... this.props} className="inngang inngang__app">
```

Comment:
For å bedre følge BEM her så ville jeg endret strukturen til (e.g bare `appinngang`-klassen) 
```jsx
<div className="appinngang" {...this.props}>
  <Innholdstittel className="appinngang__tittel">{this.props.tittel}</Innholdstittel>
</div>
```

Men for å følge opp feedback fra P2 og snakkeboblene så bør vi vurdere å bare bruke `this.props.children` fremfor å tvinge frem Typografien. Men det kommer litt an på hvilke behov komponenten er tiltenkt å dekke.